superset/assets/src/dashboard/util/charts/getEffectiveExtraFilters.js

```diff
@@ -54,7 +54,22 @@ const getFilter = (col, op, val) => {
   }
 }
 
-const valueToString = (value) => value ? value.toString() : value;
+const valueToStringErr = (value) => {
+  console.log("Error: [Prototype-less object]/ [null] encountered for getEffectiveExtraFilters");
+  return "Error: [Prototype-less object]/ [null] encountered";
+}
+
+const valueToString = (value) => {
+  return value == null ? valueToStringErr(value) : (typeof value === 'object' && !value.toString ? valueToStringErr(value) : String(value));
```

Comment:
I think for `null` we can return same as existing code i.e  `null`
```
valueToString = (value) => value ? value.toString() : value;
```

Reply:
Sure.